src/qt3utils/applications/qt3ple/main.py

```diff
@@ -0,0 +1,503 @@
+        self.downsweep_button = tk.Checkbutton(frame, text="Downsweep", onvalue=1, offvalue=0,
+                                               variable=self.do_downsweep, command=self.display_downsweep_time)
+        self.downsweep_button.grid(row=row, column=2)
+
+        row += 1
+        tk.Label(frame, text="Mode").grid(row=row, column=0)
+        self.scan_mode = tk.StringVar()
+        self.scan_mode.set("Discrete")
+        self.scan_options = tk.OptionMenu(frame, self.scan_mode, *SCAN_OPTIONS, command=self.display_batch_entry)
+        self.scan_options.grid(row=row, column=1)
+        self.batch_entry = tk.Entry(frame, width=10)
+        self.batch_entry.insert(10, 1)
+        self.batch_entry.grid(row=row, column=2)
+        self.batch_entry.config(state=tk.DISABLED)
+
+        row += 1
+        tk.Label(frame, text="DAQ Settings", font='Helvetica 16').grid(row=row, column=0, pady=10)
+
+        row += 1
+        self.goto_button = tk.Button(frame, text="Go To Voltage")
+        self.goto_button.grid(row=row, column=0)
+        self.goto_slow_button = tk.Button(frame, text="Slowly go to Voltage")
+        self.goto_slow_button.grid(row=row, column=1)
+        row += 1
+        tk.Label(frame, text="Voltage (V)").grid(row=row, column=0)
+        self.voltage_entry = tk.Entry(frame, width=10)
+        self.voltage_entry.insert(10, 0)
+        self.voltage_entry.grid(row=row, column=1)
+
+        row += 1
+        self.get_button = tk.Button(frame, text="Get current Voltage")
+        self.get_button.grid(row=row, column=0)
+        self.voltage_show=tk.Label(frame, text='None')
+        self.voltage_show.grid(row=row, column=1)
+
+        row += 1
+        tk.Label(frame, text="Voltage Limits (V)").grid(row=row, column=0)
+        self.voltage_lmin_entry = tk.Entry(frame, width=10)
+        self.voltage_lmax_entry = tk.Entry(frame, width=10)
+        self.voltage_lmin_entry.insert(10, float(scan_range[0]))
+        self.voltage_lmax_entry.insert(10, float(scan_range[1]))
+        self.voltage_lmin_entry.grid(row=row, column=1)
+        self.voltage_lmax_entry.grid(row=row, column=2)
+
+        row += 1
+        tk.Label(frame, text="Hardware Configuration", font='Helvetica 16').grid(row=row, column=0, pady=10)
+
+        row+=1
+        self.controller_option = tk.StringVar(frame)
+        self.controller_option.set(DEFAULT_DAQ_DEVICE_NAME)  # setting the default value
+
+        self.controller_menu = tk.OptionMenu(frame,
+                                             self.controller_option,
+                                             *STANDARD_CONTROLLERS.keys(),
+                                             command=root.load_controller_from_name)
+
+        self.controller_menu.grid(row=row, column=0, columnspan=3)
+
+        row += 1
+        self.hardware_config_from_yaml_button = tk.Button(frame, text="Load YAML Config")
+        self.hardware_config_from_yaml_button.grid(row=row, column=0, columnspan=3)
```

Comment:
There is no GUI for changing the yaml file settings here. Is there a plan to add one?